src/frontend/src/flows/register/captcha.ts

```diff
@@ -219,7 +219,9 @@ export const promptCaptcha = ({
     promptCaptchaPage({
       cancel: () => resolve(cancel),
       verifyChallengeChars: async ({ chars, challenge }) => {
-        const tempIdentity = Ed25519KeyIdentity.generate();
+        const tempIdentity = await ECDSAKeyIdentity.generate({
+          extractable: false,
```

Comment:
is this not the default, or are you being explicit?

Reply:
It is the default. But since this is an important property for us, I set it explicitly.